examples/metrics/index.ts

```diff
@@ -34,7 +34,7 @@ const subscription = topic.onEvent("for-each-url", async (event) => {
     }
 });
 
-const metric = aws.lambda.metrics.duration({ unit: "Seconds", dimensions: { FunctionName: subscription.func.name } });
-const alarm = metric.createAlarm("LongDuration", { threshold: 60, evaluationPeriods: 2 });
+const metric = subscription.func.metricDuration({ unit: "Seconds" });
```

Comment:
Note how there is no need ot provide 'dimensions' anymore, we do that on the user's behalf.